docs/compute-element/hosted-ce.md

```diff
@@ -0,0 +1,139 @@
+and configure it for the science communities that you choose to support.
+
+This document explains the requirements and the procuedure for obtaining a hosted CE.
+If you wish to have a local compute element instead,
+see page for [installing HTCondor-CE](/compute-element/install-htcondor-ce).
+
+![managed services diagram](/compute-element/img/managed_services_diagram.png)
+
+
+Before Starting
+---------------
+
+Before preparing your cluster for OSG jobs, consider the following requirements:
+
+-   An existing compute cluster with a [supported batch system](/index.md#prepare-the-batch-system)
+    running on a [supported operating system](/release/supported_platforms)
+
+-   Outbound network connectivity from the compute nodes (they can be behind NAT)
+
+-   A Unix account on your cluster's submit server, accessible via an SSH key
+    The OSG CE will use this account to automatically submit jobs,
+    so it must also have permissions to submit jobs to the batch system
+
+-   If your batch system is not HTCondor,
+    there must be a shared file system between the submit server and the compute nodes
+
+
+Which Science Communities and Institutions am I Supporting?
+-----------------------------------------------------------
+
+The OSG provides monitoring to view which communities are accessing your site, their fields of science, and home institution.
+Below is an example of the monitoring views that will be available for your cluster.
+
+![monitoring graphs](/compute-element/img/monitoring_graphs.png)
+
+
+Security
+--------
+
+OSG takes multiple precautions to maintain security and prevent unauthorized
+usage of resources:
+
+-   Access to the OSG system with SSH keys are restricted to the OSG staff maintaining them
+-   Users are carefully vetted before they are allowed to submit jobs to OSG
+-   Jobs running through OSG can be traced back to the user that submitted them
+-   Job submission can quickly be disabled if needed
+-   OSG staff are readily contactable in case of an emergency,
+    through email at <mailto:help@opensciencegrid.org>
+
+
+Applying for an OSG Hosted CE
+-----------------------------
+
+Before making any system changes, you should do the following steps:
+
+-   Fill out the [cluster integration questionnaire](http://goo.gl/forms/8OukxsyG6KBSGHuR2)
+    so that the OSG team has basic information about your cluster
+
+-   Email [help@opensciencegrid.org](mailto:help@opensciencegrid.org)
+    to set up a consultation call with the OSG team,
+    in order to discuss how you would like to contribute to the OSG;
+    for example, the number of OSG jobs that should run, what resource limits you have,
+    or which science communities you support
+
+After the consultation, do the following:
+
+-   Create a Unix account on the submit server for use by the OSG CE
+
+-   Install public SSH keys for the account
+
+Once this is done, the OSG team will:
+
+-   Configure the OSG CE with your system details
```

Comment:
Let's replace OSG CE with "Hosted CE" everywhere in the doc and make sure Hosted CE is capitalized consistently

Reply:
Updated